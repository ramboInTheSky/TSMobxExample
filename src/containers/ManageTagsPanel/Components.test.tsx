import React from 'react'
import { shallow } from 'enzyme'

import {
	Wrapper,
	Container,
	TitleField,
	Title,
	ProfileName,
	AddTagContainer,
	InputContainer,
	Input,
	Action,
	ActionsBar,
	ButtonsContainer,
	VisitDateFrom,
	VisitDateTo
} from './Components'

describe('<Wrapper />', () => {
	it('renders correctly', () => {
		expect(shallow(<Wrapper />)).toMatchSnapshot()
	})
})

describe('<Container />', () => {
	it('renders correctly', () => {
		expect(shallow(<Container />)).toMatchSnapshot()
	})
})

describe('<TitleField />', () => {
	it('renders correctly', () => {
		expect(shallow(<TitleField />)).toMatchSnapshot()
	})
})

describe('<Title />', () => {
	it('renders correctly', () => {
		expect(shallow(<Title />)).toMatchSnapshot()
	})
})

describe('<ProfileName />', () => {
	it('renders correctly', () => {
		expect(shallow(<ProfileName />)).toMatchSnapshot()
	})
})

describe('<AddTagContainer />', () => {
	it('renders correctly', () => {
		expect(shallow(<AddTagContainer />)).toMatchSnapshot()
	})
})

describe('<Action />', () => {
	it('renders correctly', () => {
		expect(shallow(<Action />)).toMatchSnapshot()
	})
})

describe('<InputContainer />', () => {
	it('renders correctly', () => {
		expect(shallow(<InputContainer />)).toMatchSnapshot()
	})
})

describe('<Input />', () => {
	it('renders correctly', () => {
		expect(shallow(<Input />)).toMatchSnapshot()
	})
})

describe('<ButtonsContainer />', () => {
	it('renders correctly', () => {
		expect(shallow(<ButtonsContainer />)).toMatchSnapshot()
	})
})

describe('<ActionsBar />', () => {
	it('renders correctly', () => {
		expect(shallow(<ActionsBar />)).toMatchSnapshot()
	})
})

describe('<VisitDateFrom />', () => {
	it('renders correctly', () => {
		expect(shallow(<VisitDateFrom />)).toMatchSnapshot()
	})
})

describe('<VisitDateTo />', () => {
	it('renders correctly', () => {
		expect(shallow(<VisitDateTo />)).toMatchSnapshot()
	})
})
