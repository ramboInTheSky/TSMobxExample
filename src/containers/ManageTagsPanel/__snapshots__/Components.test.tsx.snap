// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Action /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<ActionsBar /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<AddTagContainer /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<ButtonsContainer /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<Container /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<Input /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<InputContainer /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<ProfileName /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<Title /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<TitleField /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<VisitDateFrom /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<VisitDateTo /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<Wrapper /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
